README.md

```diff
@@ -84,3 +81,5 @@ It is important to understand that due to the nature of the feature, not all deb
 
 * **Is it possible to load out-of-tree kernel modules?** All the kernel modules need to be signed with a trusted key. At this moments we only sign the module at build time so only the out-of-tree modules that we build and ship as a part of `balenaOS` are properly signed. Loading user-built kernel modules require building custom software and is an extra service available on demand.
 * **Can I generate and use my own certificates and sign my own software?**: Publicly accessible balenaOS images are signed with shared balena certificates. Signing with custom certificates is possible but requires building custom software and is an extra service available on demand.
+* **How do I know my device has secure boot enabled?**: Until an API interface is available, the `imx-otp-tool is-secured` command can be used from the hostOS shell.
+* **How do I know my device is using encrypted storage?**: Dm-crypt partitions appear as `/dev/mapper` devices in the hostOS. You can use the `mount` command to check that all system partitions are mounted from `/dev/mapper` devices.
```

Comment:
Mostly for my understanding, I suppose using DM directly also creates the `/dev/dm-X` nodes, right?

Reply:
that's my understanding, yes.